terminus-terminal/src/components/appearanceSettingsTab.component.pug

```diff
@@ -109,6 +109,24 @@ h3.mb-3 Appearance
         (ngModelChange)='config.save()',
     )
 
+.form-line
+    .header
+        .title Disable Tab Index
+
+    toggle(
+        [(ngModel)]='config.store.terminal.disableTabIndex',
+        (ngModelChange)='config.save(); config.requestRestart()',
```

Comment:
Restart shouldn't be required if you bind directly to the settings in `TabHeaderComponent`